composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "joetannenbaum/phpushbullet": "~1.2",
-        "laravel/framework": "^6"
+        "laravel/framework": ">5.6"
```

Comment:
^6|^7 should be used here. I doesn't know if this package will work with an unreleased laravel version